### ISSUE TITLE
[BUG][WCAG] requests/new - No keyboard accessibility on menu or notification drawers.

### ISSUE THREAD

**Issue by j-isenberg** [reach4help/reach4help#649]
**Describe the bug**
There is no keyboard functionality on the Menu or Notification drawers. 
Keyboard functionality and navigation is essential to those using speech-to-text command, among other accessibility technologies. 
Users are incapable of navigation or option selecting without the use of a mouse. The tab function continues to cycle through the requests/new options under the drawer. 

**Please note** that this is one of the only compliance guidelines that has 0 exceptions or levels. 

https://reach4help-staging-web-client.firebaseapp.com/requests/new

**To Reproduce**
1. Navigate to /requests/new page, 
2. Tab through or click the Menu button. 
3. Attempt to tab through the Menu drawer and notice that there is no focus indicator, or functionality. 
4. Close the drawer, and attempt the same with the Notification drawer. Notice that the same issue occurs. 

**Expected behavior**
User is able to navigate and interact with every part of the page, including drawers, using universal keyboard key-strokes. 

**Desktop (please complete the following information):**
Windows 10 - Chrome 

**Additional context**
https://www.w3.org/WAI/WCAG21/quickref/#keyboard-accessible

--------
### [ZenHub Information](https://app.zenhub.com/workspaces/reach4help-5e8dcbfb14ac087f410cbabb/issues/reach4help/reach4help/649)

*This information is updated automatically. To modify it, please use ZenHub.*

--------
### [ZenHub Information](https://app.zenhub.com/workspaces/reach4help-5e8dcbfb14ac087f410cbabb/issues/reach4help/reach4help/649)

*This information is updated automatically. To modify it, please use ZenHub.*

**Comment by isma-osy** [comment]
Winggo

**Comment by PhilArsHermit** [comment]
Adding a comment, as it seems that the Keyboard navigation has been implemented, but theres an issue when the user want to navigate in the Menu.  

The user can navigate, but he cant see where the cursor is pointing.  He can see some location, but only because the url is shown at the bottom, and its only for a few section.
![Keyboard_Navigation_Menu](https://user-images.githubusercontent.com/65713824/85033970-8a5b6680-b14f-11ea-94d8-33d8c555e72e.png)

We should see the section highlighted, like when we pass the cursor over.

**Comment by mcverter** [comment]
@PhilArsHermit We actually made no specific changes to the code to address Keyboard navigation.  The reason why the Keyboard navigation is working is that we are writing our markup the right way and the markup automatically takes care of keyboard navigation.  That said, I have never been entrely clear what you mean by "Keyboard Navigation" or how you are testing for it.

@sabind @ashwinkjoseph

**Comment by PhilArsHermit** [comment]
A simple way to test it, is with a Volunteer, click on the Map icon.  The section is then highlighted in purple, and then you can navigate by pressing "Tab" to go forward, or  "Shift"Tab" to go backward.
